<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransferPane</class>
 <widget class="QWidget" name="FileTransferPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="actionRun"/>
     <addaction name="actionStop"/>
     <addaction name="separator"/>
     <addaction name="actionRetry"/>
     <addaction name="actionRetry_All"/>
     <addaction name="separator"/>
     <addaction name="actionDelete_Failed"/>
     <addaction name="actionDelete_All"/>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
  </layout>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Run_16x.svg</normaloff>:/Resource/icons/Run_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Stop_16x.svg</normaloff>:/Resource/icons/Stop_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/DeleteAllRows_16x.svg</normaloff>:/Resource/icons/DeleteAllRows_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRetry">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/CloudRefresh_16x.svg</normaloff>:/Resource/icons/CloudRefresh_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Retry</string>
   </property>
  </action>
  <action name="actionRetry_All">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/CloudStagingRefresh_16x.svg</normaloff>:/Resource/icons/CloudStagingRefresh_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Retry All</string>
   </property>
  </action>
  <action name="actionDelete_All">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Cancel_16x.svg</normaloff>:/Resource/icons/Cancel_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Delete All</string>
   </property>
   <property name="toolTip">
    <string>Delete All</string>
   </property>
  </action>
  <action name="actionDelete_Failed">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/TestCoveredException_16x.svg</normaloff>:/Resource/icons/TestCoveredException_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Delete Failed</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../r.qrc"/>
 </resources>
 <connections/>
</ui>
